solr/core/src/java/org/apache/solr/cloud/api/collections/OverseerCollectionMessageHandler.java

```diff
@@ -73,6 +69,10 @@ public class OverseerCollectionMessageHandler implements OverseerMessageHandler,
   Stats stats;
   TimeSource timeSource;
 
+  public interface Lock {
```

Comment:
Perhaps this better belongs in LockTree?

Reply:
Okay, will change